packages/utils/src/lib/reports/generate-md-reports-diff-utils.ts

```diff
@@ -0,0 +1,182 @@
+import { InlineText, MarkdownDocument, md } from 'build-md';
+import { ReportsDiff } from '@code-pushup/models';
+import { pluralize, pluralizeToken } from '../formatting';
+import { objectToEntries } from '../transform';
+import { DiffOutcome } from './types';
+
+// to prevent exceeding Markdown comment character limit
+const MAX_ROWS = 100;
+
+export function summarizeUnchanged(
+  token: string,
+  { changed, unchanged }: { changed: unknown[]; unchanged: unknown[] },
+): string {
+  return [
+    changed.length > 0
+      ? pluralizeToken(`other ${token}`, unchanged.length)
+      : `All of ${pluralizeToken(token, unchanged.length)}`,
+    unchanged.length === 1 ? 'is' : 'are',
+    'unchanged.',
+  ].join(' ');
```

Comment:
```suggestion
const unchangedCount = unchanged.length;
  const tokenDescription = changed.length > 0
    ? pluralizeToken(`other ${token}`, unchangedCount)
    : `All of ${pluralizeToken(token, unchangedCount)}`;
  const verb = unchangedCount === 1 ? 'is' : 'are';

  return `${tokenDescription} ${verb} unchanged.`;
```

Reply:
Refactored in 94fed1a2c374afeb687f4d95632ede06d84e1155.